overlord/ifacestate/ifacemgr.go

```diff
@@ -111,6 +144,36 @@ func (m *InterfaceManager) Repository() *interfaces.Repository {
 	return m.repo
 }
 
+// DisableUdevMonitor disables the instantiation of udev monitor, but has no effect
+// if udev is already created; it should be called after creating InterfaceManager, before
+// first Ensure.
+// This method is meant for tests only.
+func (m *InterfaceManager) DisableUdevMonitor() {
+	if m.udevMon != nil {
+		logger.Noticef("Udev Monitor already created, cannot be disabled")
```

Comment:
"Udev Monitor" => either "UDev monitor" or "UDevMonitor"

It also needs an "internal error: " prefix. This is not a user oriented error.

Also, if it's really just for tests, this can be a panic instead. It will be helpful to tell that we in fact have a monitor running when we think we don't.